### PR TITLE
Ca improvement

### PR DESCRIPTION
**Pull request by mvanmeerbeck**

### CODE REVIEW

File: src/command.ts [hunk 1]
```diff
@@ -73,8 +74,9 @@ program
   .option('-g, --genesis <path>', 'Absolute path to the genesis deployment definition file')
   .action(async cmd => {
     await CLI.generatePeersCredentials(cmd.config, cmd.hosts);
+    await CLI.generateOrdererCredentials(cmd.config, cmd.hosts);
     if (cmd.genesis) {
-      await CLI.generateOrdererCredentials(cmd.genesis);
+      //await CLI.generateOrdererCredentials(cmd.config, cmd.hosts);
```
Comment (mvanmeerbeck) [inline src/command.ts]: forgotten code commented ?

File: src/generators/artifacts/channelGenerator.ts [hunk 2]
```diff
@@ -202,9 +202,9 @@ orderers:
       }
 
       // update the provided channel
-       let sig= await channelClient.signConfig(anchorConfigPath);
+      let sig= await channelClient.signConfig(anchorConfigPath);
       const isUpdated = await channelClient.submitChannelUpdate(anchorConfigPath, [sig], channelName, this.network.organizations[0].mspName);
-    //  const isUpdated = await channelClient.updateChannel(channelName, this.network.organizations[0].mspName, anchorConfigPath);
+      //  const isUpdated = await channelClient.updateChannel(channelName, this.network.organizations[0].mspName, anchorConfigPath);
```
Comment (mvanmeerbeck) [inline src/generators/artifacts/channelGenerator.ts]: idem

File: src/generators/crypto/createOrgCerts.ts [hunk 3]
```diff
@@ -192,6 +197,158 @@ certificateAuthorities:
...
+          const ordererTlsPath = `${baseOrdererPath}/${orderer.fullName}/tls`;
+          await SysWrapper.copyFile(tlsCaCerts, `${ordererTlsPath}/ca.crt`);
+          await SysWrapper.createFile(`${ordererTlsPath}/server.crt`, ordererTlsCertificate);
+          await SysWrapper.createFile(`${ordererTlsPath}/server.key`, ordererTlsKey);
+        }
+      }
+      d('Register & Enroll Organization orderers done !!!');
+
+
+  */
```
Comment (mvanmeerbeck) [inline src/generators/crypto/createOrgCerts.ts]: What's going on here ?

File: src/command.ts (hunk 1 above)
Comment (sahar-fehri) [reply src/command.ts]: removed :p

File: src/generators/artifacts/channelGenerator.ts (hunk 2 above)
Comment (sahar-fehri) [reply src/generators/artifacts/channelGenerator.ts]: idem

File: src/generators/crypto/createOrgCerts.ts (hunk 3 above)
Comment (sahar-fehri) [reply src/generators/crypto/createOrgCerts.ts]: yes removed also it is not used

**Pull request merged**